src/main/scala/edu/ie3/simona/model/thermal/ThermalGrid.scala

```diff
@@ -48,27 +48,27 @@ final case class ThermalGrid(
     heatStorage: Option[ThermalStorage],
 ) extends LazyLogging {
 
-  /** Determines the state of the ThermalGrid by using the actual
+  /** Determines the state of the ThermalGrid by using the current
```

Comment:
Actually, "current" might be misleading here, since it's an operating point determined in the past...